.travis.yml

```diff
@@ -34,6 +34,7 @@ jobs:
         # Warm the symfony cache so it gets bundled with phar.
         - ./bin/acli
       script: composer box-compile
+      after_script: echo "$(curl -s --upload-file build/acli.phar https://transfer.sh/acli.phar)"
```

Comment:
Would this happen for PR commits too?

Reply:
Yes, this will run on PR commits. Which is great for issues like #518, so the reporting user can help test.

That's also why we use transfer.sh, because it requires no auth (Travis CI can't use API keys in PRs).

Reply:
What if you commit 10 different things locally and push? Will it only create the Phar based on the last pushed commit?

Reply:
If you push multiple commits at once, it will only build on the latest commit because Travis will only run once on that commit.

This runs in Travis so it can only build when Travis runs.